playbook.yaml

```diff
@@ -0,0 +1,30 @@
+---
+- name: hadoop-testing playbook
+
+  hosts: local
+
+  pre_tasks:
+    - name: orchestrate download file
+      ansible.builtin.template:
+       src: templates/env.j2
+       dest: "{{ playbook_dir }}/.env"
+
+    - name: Changing perm of ".env", adding "+x"
+      file: dest="{{ playbook_dir }}/.env" mode=a+x
+
+  tasks:
+    - name: orchestrate download file
+      ansible.builtin.template:
+       src: templates/download.j2
+       dest: "{{ playbook_dir }}/download.sh"
+
+    - name: Changing perm of "download.sh", adding "+x"
+      file: dest="{{ playbook_dir }}/download.sh" mode=a+x
+
+    - name: invoke download script
+      shell: sh {{ playbook_dir }}/download.sh
```

Comment:
Emm... let's keep the playbook to only do `render` works now

Reply:
OK.